tools/operator/testclient/operations/instantiatechaincode.go

```diff
@@ -365,7 +461,7 @@ func (i InstantiateCCUIObject) commitCCusingCLI(instantiateObject InstantiateCCU
 	if err != nil {
 		return err
 	}
-	return err
+	return nil
```

Comment:
you can replace all of this error check with just `return err` since if err is nil it will return nil already